features/scroll-markers.yml

```
@@ -0,0 +1,4 @@
+name: Scroll markers
+description: The `::scroll-marker` CSS pseudo-element is a focusable control within its ancestor scroll container's `::scroll-marker-group` that scrolls to its originating element.
+spec: https://drafts.csswg.org/css-overflow-5/#scroll-markers
+group: scrolling
```

BCD now has the relevant keys, so we need to add them here:

```suggestion
group: scrolling
compat_features:
  - css.selectors.scroll-marker
  - css.selectors.scroll-marker-group
  - css.properties.scroll-marker-group
  - css.properties.scroll-marker-group.after
  - css.properties.scroll-marker-group.before
  - css.properties.scroll-marker-group.none
```

Can you think of any other keys that would belong to this feature?

Once you've added them, I'm afraid you'll need to run the `npm run dist ...` command again :)

---

I believe that's everything for this feature, thanks!